SimTracker/TrackerHitAssociation/plugins/ClusterTPAssociationHeterogeneous.cc

```diff
@@ -191,7 +192,7 @@ void ClusterTPAssociationHeterogeneous::acquireGPUCuda(const edm::HeterogeneousE
   edm::Handle<CUDAProduct<TrackingRecHit2DCUDA>> gh;
   iEvent.getByToken(tGpuHits, gh);
   // temporary check (until the migration)
-  assert(gd->device() == cuda::device::current::get().id());
+  assert(gd->device() == cudautils::currentDevice());
```

Comment:
this is obsoleted by #409 .
I think it is easier for everybody not to apply this change here